### ISSUE TITLE
Chinese language installation by default?

### ISSUE THREAD

**Issue by leosenko** [adoptium/installer#452]
I am trying to distribute adoptium msi 'OpenJDK17U-jdk_x86-32_windows_hotspot_17.0.2_8.msi' and its 64 bit counter part. When I put it to GPO, I get the screen:
![obrazek](https://user-images.githubusercontent.com/3658579/162628588-ee50e2ed-0205-4861-882b-281fbb40a179.png)

Does it mean that the default language set in MSI is chinese? Can i modify the msi in Orca to change it to English?

**Comment by douph1** [comment]
No, Chinese is not the default language, but "Chinese language as used in Taiwan" must be the last added to the MSI:
https://github.com/adoptium/installer/blob/master/wix/Lang/LanguageList.config

Yes you can edit the MSI with Orca
https://github.com/adoptium/installer/tree/master/wix#language-with-gpo

and/or

Dont forget to uncheck "Language check"
https://github.com/adoptium/installer/issues/57
>The problem was the ,,language check". We have disabled it now and it works fine.

**Comment by lmojzis** [comment]
Hi there, 

I'm reaching out in regards to Issue #408 and #452. It has been over ten months since the last comment, and I wanted to check to see if there has been any progress on this matter. 

Has the default language been changed for the JRE installer, or has the behavior been modified to allow for the system locale to be used? As we are currently manually installing the JRE to our users, it would be helpful to know if this issue has been resolved, so that we can take advantage of the GPO Software Installation feature. 

Thank you for your time. 

Best, Lukáš
